### PR TITLE
Added the license file

### PR DESCRIPTION
**Pull request by salahuddinlali**
Added the license file

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,42 @@
+The Hippocratic License (MIT - Hippocratic)
+
+Copyright 2015-present Humanity Codes LLC
```
Comment (egillespie) [inline LICENSE]: Would you mind updating the year to 2020 instead of 2015?

**Review by egillespie — CHANGES_REQUESTED** [review]

File: LICENSE (hunk 1 above)
Comment (egillespie) [reply LICENSE]: @salahuddinlali, would you mind updating the license to reflect the new year?

Comment (salahuddinlali) [reply LICENSE]: Hello Erik, I have made the change to the file year, please review, thanks!

Comment (salahuddinlali) [reply LICENSE]: @egillespie for a notification test

**Review by egillespie — APPROVED** [review]
Looks great. Nice work! :shipit:

**Pull request merged**